hello-world-mtls/package.json

```diff
@@ -10,7 +10,7 @@
     "workflow": "ts-node src/execute-workflow.ts"
   },
   "dependencies": {
-    "temporalio": "^0.4.1"
+    "temporalio": "0.4.1"
```

Comment:
why pin?

Reply:
Because hello-world-mtls is still written using v0.4.1 syntax, it won't compile with v0.4.2.